client/src/routes/index.js

```diff
@@ -104,6 +105,7 @@ export default () => {
               component={ParticipantConfirmation}
             />
             <Route exact path={Routes.Base} component={ParticipantForm} />
+            <Route exact path={Routes.OtP} component={OTP} />
```

Comment:
Can we name this something along the lines of `confirm-interest`? I don't feel OTP is descriptive enough

Reply:
Sounds good. 👍